src/features/comments.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+import { Comment } from '../types/Comment';
+
+type Comments = {
+  comments: Comment[];
+  isLoading: boolean;
+  error: string;
+};
+
+const initialState: Comments = {
+  comments: [],
+  isLoading: false,
+  error: '',
+};
+
+export const loadComments = createAsyncThunk(
+  'comments/fetch',
+  async (postId: number) => {
+    return getPostComments(postId);
+  },
+);
+
+export const addNewComment = createAsyncThunk(
+  'comments/addComment',
+  async (comment: Omit<Comment, 'id'>) => {
+    return createComment(comment);
+  },
+);
+
+export const removeComment = createAsyncThunk(
+  'comments/removeComment',
+  (commentId: number) => {
+    deleteComment(commentId);
+
+    return commentId;
```

Comment:
This function should be asynchronous since it involves an API call. Consider adding the 'async' keyword before the function parameter and using 'await' before 'deleteComment(commentId)'.